src/index.js

```diff
@@ -1 +1,25 @@
-import './styles/styles.css';
\ No newline at end of file
+import './styles/styles.css';
+
+import getData from './module/data.js';
+
+const card = document.querySelector('.list-items .card');
+getData().then((data) => {
+  data.meals.forEach((item) => {
+    card.innerHTML += `
+    <li>
+          <div class="card-image">
+            <img src="${item.strMealThumb}" alt="">
+          </div>
+          <div class="card-title">
+            <span><h2>${item.strMeal}</h2></span>
+            <span class="material-symbols-outlined">favorite</span>
+            <span>5 likes</span>
+          </div>
+          <div class="card-footer">
+            <span>Comments</span>
+            <span>Reservations</span>
```

Comment:
- [ ] It is Considered Good Practice to make these Elements Should Be Buttons to address event Listeners.